pylibs/otns/cli/OTNS.py

```diff
@@ -425,9 +434,9 @@ def get_ipaddrs(self, nodeid: int, addrtype: str = None) -> List[str]:
     def set_network_name(self, nodeid: int, name: str = None):
```

Comment:
Not touched by this PR, but `get_network_name` at below is returning nothing.

Reply:
fixed, thanks